cookbooks/tian_parameterization_kinematic_slab/coupled-two-phase-tian-parameterization-kinematic-slab.prm

```diff
@@ -0,0 +1,295 @@
+# This cookbook simulates a kinematically-driven 2D subduction system containing
+# distinct lithologies for the mantle wedge and subducting sediment,
+# mid-ocean ridge basalt (MORB), gabbro, peridotite. Lithologies within the slab
+# are prescribed an initial hydration state after a period of equilibration, and
```

Comment:
are prescribed to an initial hydration state? prescribe an initial hydration state?

Reply:
I think the text as currently written makes sense?